### PR TITLE
Mft dcs processor

### PR DESCRIPTION
**Pull request by syano0822**
The PR is the first attempt to implement DCS processor.
The codes are contained in Detector/ITSMFT/MFT/condition.
For the test, mimic DCS DPs are generated with "o2-condition-mft-dcs-sim-workflow”.
The mimic generator generates Current.Analog, Current.Digital, Current.BackBias DPs within 0 - 3.
The DPs are packed into CCDB object with “o2-condition-mft-dcs-workflow”.
The codes, Detectors/ITSMFT/MFT/condition/testWorkflow/MFTDCSDataProcessorSpec.h, Detectors/ITSMFT/MFT/condition/src/MFTDCSProcessor.cxx, and Detectors/ITSMFT/MFT/condition/include/MFTCondition/MFTDCSProcessor.h, are the core code for the DCS processor workflow.

### CODE REVIEW

**Comment by peressounko** [comment]
Hi, why did you remove file  Detectors/PHOS/base/files/alignment.root from OTHER detector? Please restore it.

File: GPU/GPUTracking/utils/.gitattributes [hunk 1]
```diff
@@ -1 +0,0 @@
-/callvc.bat binary
```
Comment (shahor02) [inline GPU/GPUTracking/utils/.gitattributes:1]: @syano0822 also this file seems to be unrelated to your PR, could you please restore it?

**Review by shahor02 — CHANGES_REQUESTED** [review]
Hi @syano0822 

The DCS code looks fine (though the clang-format complains, please fix it), but there are many stray modifications, particularly, in your 1st commit you delete Alice mag.field parametrization + PHOS data file :). Since needed and stray modifications are in the same commit, will be difficult to repair it. I would propose to do something like this (I see you have many merges, please make sure that your pull is using rebase and not merge, by doing once: `git config --global pull.rebase true`
````
cd ~/alice/O2
git checkout mft-dcs-processor
cp -r  Detectors/ITSMFT/MFT/condition ~/Downloads; # save your new  directory
cp  Detectors/ITSMFT/MFT/CMakeLists.txt  ~/Downloads; # and the only modified file
git checkout dev
git pull
# in case your dev is not in sync with orgin/dev, do `git reset --hard origin/dev`
git branch -m mft-dcs-processor   mft-dcs-processorOLD # rename existing bad branch
git checkout -b  mft-dcs-processor  # create new branch with the same name
cp -r  ~/Downloads/condition  Detectors/ITSMFT/MFT/  # copy back new dir

git add condition
git commit -a -m "MFT dcs processor"
````

Then apply `git clang-format`, squash the fixup commit with with real not and forcepush.

**Comment by syano0822** [comment]
Dear peressounko and shahor02,

Thank you for your help.
This is the first direct contribution to O2 development, so I didn't understand its process.

I would like to open a new PR, so please stay tune.
I think the new one will recover the files you suggested.
However, I don't understand the "git clang-format".
Could you tell me the detail or document explaining git clang-format?

Thanks!

**Pull request closed without merging**